src/main/flight/mixer_tricopter.c

```diff
@@ -571,7 +571,7 @@ static void tailTuneModeThrustTorque(struct thrustTorque_t *pTT, const bool isTh
     case TT_WAIT_FOR_DISARM:
         if (!ARMING_FLAG(ARMED))
         {
-            float averageServoAngle = pTT->servoAvgAngle.sum / 10.0f / pTT->servoAvgAngle.numOf;
+            float averageServoAngle = pTT->servoAvgAngle.sum / 10.0f / pTT->servoAvgAngle.numOf - 90.0f;
             if (averageServoAngle > 0.5f && averageServoAngle < 40.f) { // TODO: Too wide? or totally unnecessary?
```

Comment:
Could you just for clarity use > 90.5f here and remove the - 90.0f on the line above. What happens with values [90, 90.5]? The max limit can be increased to 90+50 (TRI_TAIL_SERVO_MAX_ANGLE).